tutorials/terraform-asm-in-cluster.md

```diff
@@ -0,0 +1,445 @@
+          value       = module.gke.service_account
+        }
+        EOF
+
+        envsubst < main.tf_tmpl > main.tf
+
+1. Initialize Terraform and apply the configurations:
+
+        ${TERRAFORM_CMD} init
+        ${TERRAFORM_CMD} plan
+        ${TERRAFORM_CMD} apply -auto-approve
+
+## Configure access to your cluster
+
+1.  Connect to the GKE cluster:
+
+        gcloud container clusters get-credentials ${CLUSTER_1} --zone ${CLUSTER_1_ZONE}
+
+    Remember to unset your `KUBECONFIG` variable when you're finished.
+
+1.  Rename the cluster context for easy switching:
+
+        kubectl ctx ${CLUSTER_1}=gke_${PROJECT_ID}_${CLUSTER_1_ZONE}_${CLUSTER_1}
+
+1.  Confirm the cluster context:
+
+        kubectl ctx
+    
+    The output is similar to the following:
+
+        gke-central
+
+## Deploy the Online Boutique app
+
+1.  Set the Anthos Service Mesh revision variable:
+
+        export ASM_REVISION=${ASM_REV}
+    
+1.  Deploy the Online Boutique app to the GKE cluster:
+
+        kpt pkg get \
+          https://github.com/GoogleCloudPlatform/microservices-demo.git/release \
+          online-boutique
+
+        kubectl --context=${CLUSTER_1} create namespace online-boutique
+        kubectl --context=${CLUSTER_1} label namespace online-boutique istio.io/rev=${ASM_REVISION}
+        kubectl --context=${CLUSTER_1} -n online-boutique apply -f online-boutique
+
+1.  Wait until all Deployments are ready:
+
+        kubectl --context=${CLUSTER_1} -n online-boutique wait --for=condition=available --timeout=5m deployment adservice
+        kubectl --context=${CLUSTER_1} -n online-boutique wait --for=condition=available --timeout=5m deployment checkoutservice
+        kubectl --context=${CLUSTER_1} -n online-boutique wait --for=condition=available --timeout=5m deployment currencyservice
+        kubectl --context=${CLUSTER_1} -n online-boutique wait --for=condition=available --timeout=5m deployment emailservice
+        kubectl --context=${CLUSTER_1} -n online-boutique wait --for=condition=available --timeout=5m deployment frontend
+        kubectl --context=${CLUSTER_1} -n online-boutique wait --for=condition=available --timeout=5m deployment paymentservice
+        kubectl --context=${CLUSTER_1} -n online-boutique wait --for=condition=available --timeout=5m deployment productcatalogservice
+        kubectl --context=${CLUSTER_1} -n online-boutique wait --for=condition=available --timeout=5m deployment shippingservice
+        kubectl --context=${CLUSTER_1} -n online-boutique wait --for=condition=available --timeout=5m deployment cartservice
+        kubectl --context=${CLUSTER_1} -n online-boutique wait --for=condition=available --timeout=5m deployment loadgenerator
+        kubectl --context=${CLUSTER_1} -n online-boutique wait --for=condition=available --timeout=5m deployment recommendationservice
+
+## Access the Online Boutique app
+
+Access the app using the `istio-ingressgateway` Service hostname:
```

Comment:
The same step in the next document in the series uses this wording:
"Run the following command to get the IP address of the external load balancer:"

Reply:
Updated docs to use consistent language